src/wallet/wallet2.cpp

```diff
@@ -6282,8 +6282,10 @@ bool wallet2::save_multisig_tx(const std::vector<pending_tx>& ptx_vector, const
   return epee::file_io_utils::save_string_to_file(filename, ciphertext);
 }
 //----------------------------------------------------------------------------------------------------
-bool wallet2::load_multisig_tx(cryptonote::blobdata s, multisig_tx_set &exported_txs, std::function<bool(const multisig_tx_set&)> accept_func)
+
+bool wallet2::parse_multisig_tx_from_str(const std::string &multisig_tx_st, multisig_tx_set &exported_txs) const
 {
+  std::string s = multisig_tx_st;
```

Comment:
Can't you just pass std::string &multisig_tx_st ?

Reply:
like that?